development/modules/students/servicesStudents.js

```diff
@@ -36,9 +37,62 @@
       return Student.logout()
     }
 
+    function CommentStudentRelation (student) {
+      this.basePath = originsManager.getOrigin() +
+        "/" + ROUTES.STUDENTS.__BASE__ +
+        "/" + student.id +
+        "/" + ROUTES.STUDENTS.COMMENTS;
+
+    }
+
+    CommentStudentRelation.prototype = new Array();
+    CommentStudentRelation.prototype.getComments = function () {
+      return $http.get(this.basePath)
+        .then(function (response) {
+          response.data.forEach(function (comment) {
+
+            comment.publication = {
+              active: false,
+              isPublicationLoaded: false
+            };
+            comment.getPublication = function () {
+              if (!comment.publication.isPublicationLoaded) {
+                var path = originsManager.getOrigin() +
+                  "/" + ROUTES.PUBLICATIONS.__BASE__ +
+                  "/" + comment.publication_id;
+                $http.get(path)
+                  .error(function (error) {
```

Comment:
Error is not an ES Promises standard method. Please use then and catch at Promises.